### PR TITLE
Sharing with a link

### PR DESCRIPTION
**Pull request by schlosser**
The sharing by link button on the left

screen shot 2013-11-17 at 2 39 20 pm

### CODE REVIEW

**Comment by brishin** [comment]
Can you merge with master? Looks like there are conflicts.

**Comment by schlosser** [comment]
Fixed conflicts.

File: src/js/directives/courseBlock.coffee [hunk 1]
```diff
@@ -4,12 +4,20 @@ angular.module('Courses.directives')
   restrict: 'E'
   scope:
     schedule: '='
+  controller: (
+    $scope
```
Comment (brishin) [inline src/js/directives/courseBlock.coffee]: Technically you should be using commas here, but I guess Coffeescript takes the \n as a comma.

**Comment by brishin** [comment]
Approving post-nits

**Pull request merged**